projects/ngrx.io/content/examples/testing-store/src/app/reducers/auth.reducer.ts

```diff
@@ -0,0 +1,33 @@
+import { Action } from '@ngrx/store';
+import { AuthActions, AuthActionTypes } from '../auth.actions';
+
+export interface State {
+  loggedIn: boolean;
+}
+
+export const initialState: State = {
+  loggedIn: false,
+};
+
+export function reducer(
```

Comment:
Use `createReducer` function here

Reply:
is `createReducer` available in a released beta version?